package.json

```diff
@@ -2,14 +2,21 @@
   "name": "single-spa-vue",
   "version": "1.10.1",
   "description": "a single-spa plugin for vue.js applications",
-  "main": "lib/single-spa-vue.js",
+  "main": "dist/umd/single-spa-vue.js",
+  "module": "dist/esm/single-spa-vue.js",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/single-spa-vue.js",
+      "require": "./dist/umd/single-spa-vue.js"
+    }
+  },
   "scripts": {
     "prepublishOnly": "yarn build",
-    "build": "rimraf lib && rollup -c",
+    "build": "rimraf dist && rollup -c",
```

Comment:
The output files are now in the dist directory, which makes a bit more sense than `lib` i think for a project with multiple distributables.